### PR TITLE
WRO-11514: Add Repository/Hompage address to templates packages at npmjs.com

### PR DESCRIPTION
**Pull request by sjro**
Enact-DCO-1.0-Signed-off-by: SJ RO (sj.ro@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Homepage information on @enact/template-* of npmjs.com is incorrect.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Add 'homepage' field to package.json according to the guide of https://docs.npmjs.com/cli/v8/configuring-npm/package-json

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-11514

### CODE REVIEW

**Review by hong6316 — APPROVED** [review]
LGTM

**Comment by MikyungKim** [comment]
Please update the ticket number in title and description.

File: package.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "description": "A collection of Enact-based templates.",
   "private": true,
   "author": "Jason Robitaille <jason.robitaille@lge.com>",
-  "repository": "https://github.com/enactjs/templates",
```
Comment (MikyungKim) [inline package.json:7]: Let's just keep this.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]
LGTM

**Pull request merged**